modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -70,8 +70,27 @@ platform:
 publish: External
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed12345 AAAA...
+compute:
+  platform:
+    ovirt:
+      ..
```

Comment:
My understanding is that usually three dots are used to denote addition unseen content in a code block. You could update `..` to `...` in this line and the other relevant lines in this PR.

Reply:
Please see my answer to @pneedle-rh 's comment above

Reply:
The dots in this case are supposed to denote additional content in this particular block that could be there in the platform.ovirt section, for example cpu or vmType (see Example Custom machine pools in an install-config.yaml file section in this file). So, the more I think of this, the more I'm convinced this should actually be included in that section, unless there's a specific reason to have it here. 
Maybe @janosdebugs knows more or knows who to ask?